apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -152,6 +153,52 @@ export const siteRouter = router({
 
       return updatedConfig
     }),
+  updateSiteIntegrations: protectedProcedure
+    .input(updateSiteIntegrationsSchema)
+    .mutation(async ({ ctx, input: { siteId, data } }) => {
+      await validateUserPermissionsForSite({
+        siteId,
+        userId: ctx.user.id,
+        action: "update",
+      })
+
+      const {
+        user: { id: userId },
+      } = ctx
+
+      const user = await db
+        .selectFrom("User")
+        .where("id", "=", userId)
+        .selectAll()
+        .executeTakeFirstOrThrow()
```

Comment:
minor minor ask: can we just use `ctx.user.id` directly in line 171? so its consistent with other part of the codebase 😬